docs/main/pihole-command.md

```diff
@@ -198,9 +198,9 @@ Display the running status of Pi-hole's DNS and blocking services.
 
 | | |
 | -------------- | -------------- |
-| Help Command    | `pihole disable --help` |
+| Help Command    | `pihole disable --help`/`pihole enable --help` |
 | Script Location | [`/usr/local/bin/pihole`](https://github.com/pi-hole/pi-hole/blob/master/pihole) |
-| Example Usage   | [`pihole disable 5m`](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#toggle) |
+| Example Usage   | [`pihole disable 5m`/`pihole enable`](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#toggle) |
```

Comment:
```suggestion
| Example Usage   | [`pihole disable 5m`/`pihole enable 5m`](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#toggle) |
```

With v6 you can also set a timer of enabling blocking

Reply:
I know but I left it out on purpose TBH, because I assume we want it enabled forever. Otherwise the user can look at `pihole enable --help`